vllm/compilation/fix_functionalization.py

```diff
@@ -32,6 +33,10 @@ def __call__(self, graph: torch.fx.Graph):
         self.nodes_to_remove: list[torch.fx.Node] = []
         count = 0
         for node in graph.nodes:
+            # XPU does not support auto-functionalization yet.
+            # Will enable this when switch to vllm-xpu-kernels.
+            if current_platform.is_xpu():
+                continue
```

Comment:
I'm confused: what does it mean that XPU does not support autofunctionalization? And if so we should disable it at the pass level, not inside the loop

Reply:
my understanding is autofunctionalization pass is pretty special, not like some rule/op base pass, this  will always be called in current code path. I feel it will be strange to change that. 
I move this part to eariler. please take a review again.